Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:4.1
+// swift-tools-version:4.0
```

Comment:
Is this not meant to be 4.2?

Reply:
No - It supports both Swift 4 and Swift 4.2.

Reply:
Apologies. I must be doing something wrong when pulling in 0.9.0 binary via Carthage. Throws `Incompatible Swift version - framework was built with 4.0.3 ...`

Reply:
Uhm that shouldn't be the case, I built it locally on my mac which has Swift 4.2. Keep trying and if you don't figure this out by weekend I'll try to help out with local testing.